api/v1alpha1/jobset_types.go

```diff
@@ -47,6 +47,13 @@ type JobSetSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	ReplicatedJobs []ReplicatedJob `json:"replicatedJobs,omitempty"`
 
+	// SuccessPolicy, if set, configures when to declare the JobSet as
```

Comment:
```suggestion
	// SuccessPolicy configures when to declare the JobSet as
```

Regardless of it being set you apply the defaults of this?

Reply:
done